lib/components/dropdown-item.vue

```diff
@@ -1,31 +1,21 @@
 <template>
     <b-link class="dropdown-item"
             role="menuitem"
```

Comment:
When using the button, the dropdown doesn't close. I can't seem to find the logic around that. I thought the clickOutListener might be doing it, but it explicitly avoids running when triggered from an element inside the dropdown.